schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
     date_of_birth DATE,
     escape_attempts INT,
     neutered boolean,
-    weight_kg DECIMAL
+    weight_kg DECIMAL,
+    species VARCHAR(100)
```

Comment:
- [ ] You should `alter` the table instead of changing create syntax. (Table is already created on the previous milestones)
For example:
```sql
   ALTER TABLE animals ADD species varchar(255);
```

Reply:
Thank you for reminding me about that! I will fix it right away! 😊